### DISCUSSION TITLE
embedded context

### DISCUSSION THREAD

**Discussion by EnariSaarBolt** [NVIDIA/MatX#500] [Q&A] [answered]
Has someone tested the effectiveness in an embedded linux/baremetal context?

**Comment by cliffburdick** [comment, marked as answer]
Hi @EnariSaarBolt, we have run the code on Jetson Xavier products and it works great as long as you're on JetPack 5.0 to give you the minimum CUDA requirements. Whether you're in bare metal or a container should give the same performance.
